java/ehcache/Portfile

```diff
@@ -21,33 +26,29 @@ long_description	Ehcache is a pure Java, in-process cache with the following fea
 	10.	Available under the Apache 1.1 license. Ehcache's copyright and licensing has been reviewed and approved by the Apache Software Foundation, making ehcache suitable for use in Apache projects. \
 	11.	Production tested. All final versions of ehcache are production tested for several weeks on a large and very busy eCommerce site before release.
 
-homepage			http://ehcache.sourceforge.net/
+extract.suffix		.jar
+extract.only
 
-master_sites		sourceforge
+supported_archs 	noarch
 
-extract.suffix		.tgz
-worksrcdir			${name}
-fetch.type			cvs
-cvs.module			${name}
-cvs.root			:pserver:anonymous@cvs.sourceforge.net:/cvsroot/ehcache
-cvs.tag				RELEASE_[string map { . _ } ${version}]
-
-checksums			md5 1cc46d376838b9d4a99b1a83ebabb88c
-
-depends_build		bin:ant:apache-ant
-depends_lib			bin:java:kaffe
+checksums			md5 a6f66597f5aca8104972a79e72d62a04 \
```

Comment:
Do you really need `md5`? It's not strictly deprecated. If the upstream publishes `md5` it may be kept.

Reply:
Following the other existing portfiles, I had wished to retain it. I may remove it, may please let me know if required.